### PR TITLE
fix text2embedding 

### PR DESCRIPTION
**Pull request by elephantmipt**
## Description

Just small fix. In considering we are using `nn.DataParallel` we should call `.config` through `model.module.config`

## Related Issue

https://github.com/catalyst-team/catalyst/issues/700

## Type of Change

- [ ] Examples / docs / tutorials / contributors update
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Improvement (non-breaking change which improves an existing feature)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] I have read the [Code of Conduct](https://github.com/catalyst-team/catalyst/blob/master/CODE_OF_CONDUCT.md) document.
- [x] I have read the [Contributing](https://github.com/catalyst-team/catalyst/blob/master/CONTRIBUTING.md) guide.
- [x] I have checked the code-style using `make check-codestyle`.
- [x] I have written tests for all new methods and classes that I created.
- [ ] I have written the docstring in Google format for all the methods and classes that I used.
- [ ] I have checked the docs using `make check-docs`.
- [x] I have read I need to click 'Login as guest' to see Teamcity build logs.

### CODE REVIEW

**Comment by Scitator** [comment]
hi, @PUSSYMIPT 
Thanks for contributing, could you please add some small test for this contrib script?
for example, use the data from [bert classification example](https://github.com/catalyst-team/catalyst/blob/master/examples/distilbert_text_classification/input/train.csv)?
something like,
```
catalyst-contrib text2embedding --in-cvs=./examples/distilbert_text_classification/input/train.csv
```
to the end of our nlp testing script?
https://github.com/catalyst-team/catalyst/blob/master/bin/tests/check_dl_nlp.sh

**Comment by elephantmipt** [comment]
I'm not sure that `--in-huggingface` is a good idea, but at least it is more convenient to write the test without downloading all bert's weights with `wget`

File: catalyst/data/scripts/text2embedding.py [hunk 1]
```diff
@@ -140,10 +169,21 @@ def main(args, _=None):
             mask = batch["attention_mask"].unsqueeze(-1) \
                 if args.mask_for_max_length \
                 else None
-            features_ = utils.process_bert_output(
+
+            if isinstance(model, torch.nn.DataParallel):
```
Comment (Scitator) [inline catalyst/data/scripts/text2embedding.py]: could you please use `utils.is_wrapped_with_???` as more convenient way?

**Review by Scitator — CHANGES_REQUESTED** [review]

**Comment by Scitator** [comment]
@PUSSYMIPT thank you for your Pull Request, everything looks good.
Please merge with master branch and let's merge it out!

**Comment by Scitator** [comment]
@PUSSYMIPT Thank you for your contribution!

**Pull request merged**